docs/pages/build/building-from-github.mdx

```diff
@@ -59,25 +59,28 @@ page](https://expo.dev/accounts/[account]/projects/[projectName]/github).
 
 Install the Expo GitHub App on your GitHub account.
 
+> **Note:** You must be an admin of the Expo account to install the app.
```

Comment:
```suggestion
> **Note:** You must have [Owner or Admin access](/accounts/working-together/#manage-access) of the Expo account to install the app.
```

Let's link to Owner and Admin role and also specify Owner and Admin both so as not leave any room for confusion.